### PR TITLE
kmag: init at 19.12.3

### PR DESCRIPTION
**Pull request by SRGOM**
###### Motivation for this change
Towards https://github.com/NixOS/nixpkgs/issues/62168 

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/kde/kmag.nix [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+, extra-cmake-modules
+, kdoctools
+, kconfig, kio, ktextwidgets, kxmlgui
+}:
+
+mkDerivation {
+  name = "kmag";
+  nativeBuildInputs = [ extra-cmake-modules ];
+  buildInputs = [ kdoctools kxmlgui kio ];
+  meta = {
```
Comment (Thra11) [inline pkgs/applications/kde/kmag.nix:13]: How about `description = "A screen magnifier for KDE";`

File: pkgs/applications/kde/kmag.nix [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    license = with lib.licenses; [ gpl2 ];
+    maintainers = with lib.maintainers; [ ];
```
Comment (ttuegel) [inline pkgs/applications/kde/kmag.nix:15]: Please add yourself to `maintainers`.

**Comment by papojari** [comment]
kmag was already added.

**Comment by papojari** [comment]
@ttuegel can you please close this?

**Pull request closed without merging**